libs/styles/_texts.scss

```
@@ -0,0 +1,39 @@
+.contler-h1 {
```

```suggestion
.ctl-h1 {
```
use ctl for Contler abbreviation